src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderBooking.java

```diff
@@ -255,7 +240,7 @@ public class OffenderBooking extends AuditableEntity {
     @Default
     @Exclude
     @BatchSize(size = 1000)
-    private List<OffenderSentence> sentences = new ArrayList<>();
+    private Set<OffenderSentence> sentences = new HashSet<>();
```

Comment:
fix to prevent duplicates coming from using fetch joins

Reply:
So simple and yet so painful to find